src/PythonMigrationViewExtension/MigrationAssistant/ScriptMigrator.cs

```diff
@@ -39,7 +39,7 @@ internal static string MigrateCode(string code)
                         scope.Exec(GetPythonMigrationScript());
 
                         var result = scope.Contains(RETURN_NAME) ? scope.Get(RETURN_NAME) : null;
-                        return result.ToString();
+                        return result?.ToString();
```

Comment:
should this return an empty string if result is null?

Reply:
note : result?.ToString() ?? string.Empty;

Reply:
var result = scope.Contains(RETURN_NAME) ? scope.Get(RETURN_NAME) : string.Empty;